src/styles/blocks/review.css

```diff
@@ -0,0 +1,16 @@
+.review {
+  align-self: flex-start;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 16px;
```

Comment:
do not hardcode the height, let the content dictate it

Reply:
Done